src/drivers/mct/bld/testdir/env_mach_pes.xml

```diff
@@ -0,0 +1,324 @@
+      <desc>root wav mpi task</desc>
+    </entry>
+    <entry id="NINST_WAV" value="1">
+      <type>integer</type>
+      <desc>Number of wave instances</desc>
+    </entry>
+    <entry id="NINST_WAV_LAYOUT" value="concurrent">
+      <type>char</type>
+      <valid_values>sequential,concurrent</valid_values>
+      <desc>Layout of wave instances</desc>
+    </entry>
+  </group>
+  <group id="mach_pes_esp">
+    <entry id="NTASKS_ESP" value="1">
+      <type>integer</type>
+      <desc>number of external processing tool mpi tasks</desc>
+    </entry>
+    <entry id="NTHRDS_ESP" value="0">
+      <type>integer</type>
+      <desc>number of external processing tool mpi threads</desc>
+    </entry>
+    <entry id="ROOTPE_ESP" value="0">
+      <type>integer</type>
+      <desc>root ocn mpi task</desc>
+    </entry>
+    <entry id="NINST_ESP" value="1">
+      <type>integer</type>
+      <desc>Number of external processing tool instances</desc>
+    </entry>
+    <entry id="NINST_ESP_LAYOUT" value="concurrent">
+      <type>char</type>
+      <valid_values>sequential,concurrent</valid_values>
+      <desc>Layout of external processing tool instances</desc>
+    </entry>
+  </group>
+  <group id="mach_pes_stride">
+    <entry id="PSTRID_ATM" value="1">
+      <type>integer</type>
+      <desc>stride of mpi tasks for atm comp - currently should always be set to 1</desc>
+    </entry>
+    <entry id="PSTRID_LND" value="1">
+      <type>integer</type>
+      <valid_values>1</valid_values>
+      <desc>stride of mpi tasks for lnd comp - currently should always be set to 1</desc>
+    </entry>
+    <entry id="PSTRID_ICE" value="1">
+      <type>integer</type>
+      <desc>stride of mpi tasks for ice comp - currently should always be set to 1</desc>
+    </entry>
+    <entry id="PSTRID_OCN" value="1">
+      <type>integer</type>
+      <valid_values>1</valid_values>
+      <desc>stride of mpi tasks for ocn comp - currently should always be set to 1</desc>
+    </entry>
+    <entry id="PSTRID_CPL" value="1">
+      <type>integer</type>
+      <desc>stride of mpi tasks for cpl comp - currently should always be set to 1</desc>
+    </entry>
+    <entry id="PSTRID_GLC" value="1">
+      <type>integer</type>
+      <valid_values>1</valid_values>
+      <desc>stride of mpi tasks for glc comp - currently should always be set to 1</desc>
+    </entry>
+    <entry id="PSTRID_ROF" value="1">
+      <type>integer</type>
+      <desc>stride of mpi tasks for rof comp - currently should always be set to 1</desc>
+    </entry>
+    <entry id="PSTRID_WAV" value="1">
+      <type>integer</type>
+      <valid_values>1</valid_values>
+      <desc>stride of mpi tasks for wav comp - currently should always be set to 1</desc>
+    </entry>
+    <entry id="PSTRID_ESP" value="1">
+      <type>integer</type>
+      <valid_values>1</valid_values>
+      <desc>stride of mpi tasks for esp comp - currently should always be set to 1</desc>
+    </entry>
+  </group>
+</file>
```

Comment:
see above comment for env_build.xml - this is the wrong schema for the driver xml files and seems like a historical file that is no longer needed.